wasm/src/lib.rs

```diff
@@ -196,6 +199,28 @@ extern "C" {
     pub fn log(s: &str);
 }
 
+#[macro_export]
+macro_rules! array {
+        ($($value:expr),*$(,)?) => {{
+            let array = Array::new();
+
+            $(array.push(&JsValue::from($value));)*
+
+            array
+        }};
+    }
+
+#[macro_export]
+macro_rules! object {
+        ($($key:literal: $value:expr,)*) => {{
+            let object = Object::new();
```

Comment:
Similarly, this should use `::js_sys::Object::new()` and `::js_sys::Reflect::set` and `::wasm_bindgen::JsValue::from_str` and `::wasm_bindgen::JsValue::from`